src/dsp_tools/utils/xml_upload.py

```diff
@@ -1041,9 +1041,12 @@ def _handle_upload_error(
     Returns:
         None
     """
-
-    print("\n==========================================\nxmlupload must be aborted because of an error")
-    logger.info("xmlupload must be aborted because of an error")
+    print(
+        f"\n==========================================\n"
+        f"xmlupload must be aborted because of an error.\n"
+        f"Error message: '{err}'\n"
+    )
+    logger.error("xmlupload must be aborted because of an error", exc_info=err)
```

Comment:
print error message here, instead of re-raising the error at the end. That's much more practical.